rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -569,70 +701,83 @@ extern "C" RocalTensor ROCAL_API_CALL rocalCropMirrorNormalize(RocalContext cont
+                                                       float crop_pos_y,
+                                                       float crop_pos_z,
+                                                       RocalTensorLayout rocal_tensor_output_layout = ROCAL_NHWC,
+                                                       RocalTensorOutputType rocal_tensor_output_datatype = ROCAL_UINT8);
 // //// \param crop_width
 
 
 extern "C" RocalTensor  ROCAL_API_CALL rocalCropCenterFixed(RocalContext context, RocalTensor input,
-                                                        unsigned crop_width,
-                                                        unsigned crop_height,
-                                                        unsigned crop_depth,
-                                                        bool output);
-
-extern "C"  RocalTensor  ROCAL_API_CALL rocalResizeCropMirrorFixed( RocalContext context, RocalTensor input,
-                                                           unsigned dest_width, unsigned dest_height,
-                                                            bool is_output,
-                                                            unsigned crop_h,
-                                                            unsigned crop_w,
-                                                            RocalIntParam mirror
-                                                            );
-extern "C"  RocalTensor  ROCAL_API_CALL rocalResizeCropMirror( RocalContext context, RocalTensor input,
-                                                           unsigned dest_width, unsigned dest_height,
-                                                            bool is_output, RocalFloatParam crop_height = NULL,
-                                                            RocalFloatParam crop_width = NULL, RocalIntParam mirror = NULL
-                                                            );
+                                                            unsigned crop_width,
+                                                            unsigned crop_height,
+                                                            unsigned crop_depth,
+                                                            bool output,
+                                                            RocalTensorLayout rocal_tensor_output_layout = ROCAL_NHWC,
+                                                            RocalTensorOutputType rocal_tensor_output_datatype = ROCAL_UINT8);
+
+extern "C"  RocalTensor  ROCAL_API_CALL rocalResizeCropMirrorFixed(RocalContext context, RocalTensor input,
+                                                                   unsigned dest_width, unsigned dest_height,
+                                                                   bool is_output,
+                                                                   unsigned crop_h,
+                                                                   unsigned crop_w,
```

Comment:
Check if we can rename crop_h, crop_w to crop_height, crop_width to make it consistent with dest_width, dest_height

Reply:
I think crop_h and crop_w is understandable